src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/ReturnCountCheck.java

```diff
@@ -227,10 +234,12 @@ private void visitReturn(DetailAST ast) {
         }
 
         /**
-         * Increase the number of return statements.
+         * Increase the number of return statements and set method return type.
          * @param maxAssigned Maximum allowed number of return statements.
+         * @param voidReturn Identifies if context (method/lambda/constructor) returns anything.
```

Comment:
Please use `Identifies if context is void.`

Reply:
done